ONSdigital/census-rm-docker-dev
# Remove all references to unused docker network

# Motivation and Context
The docker network was completely unused and being automatically removed from every time I ran a docker prune because of this.

# What has changed
* Remove unused network

# How to test?
Run a `make pull up` and then run the acceptance tests, no functionallity should be impacted

# Links
https://trello.com/c/i5FMuQOn/858-remove-unused-docker-network-from-cenus-rm-docker-dev-2
